ontology/yaml/resources/HVAC/entity_types/VLV.yaml

```diff
@@ -15,6 +15,13 @@
 ########################
 ### Canonical Types  ###
 ########################
+VLV_HWZTC:
```

Comment:
This doesn't look like zone temp monitoring, this looks like zone temp control (because you are implementing HWZTC) -- if it's simply a heating water valve that modulates independent of zone temp then I would implement both HWVM and ZTM.

If the valve is actually controlling to a zone temp setpoint, then it's probably part of a large piece of equipment like a terminal unit or FCU?

Reply:
@cstirdivant - apologies it should be zone temp control, it is for a radiator valve. Have changed the description to reflect.

Reply:
I am still confused by this model -- what does this valve represent? It is a stand-alone valve that controls to a zone temp setpoint? Or is it a valve that is a part of a larger piece of equipment?

Reply:
@cstirdivant - this is a standalone valve that controls a stairwell/lobby area. It closes/opens based on the temperature setpoint.